cli/test/org/partiql/cli/functions/ReadFileTest.kt

```diff
@@ -171,12 +171,12 @@ class ReadFileTest {
 
     @Test
     fun readPostgreSQLCsvFile() {
-        writeFile("simple_postgresql.csv", "id,name,balance\n1,Bob,10000.00")
+        writeFile("simple_postgresql.csv", "id,name,balance\n1,B\"\"ob,10000.00")
```

Comment:
Since double quotes is escape in postgre_sql format, we need 2 double quotes as the first one escape the second.